### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by willtarax** [Wimpingego/nnow#11]
Wiki is sadly devoid of information. I would love to see both recipes and additional villager uses.

**Comment by Wimpingego** [comment]
Yes sadly it is, just can't seem to find the time to get it all made up.
